docs/available_commands.md

```diff
@@ -211,6 +211,41 @@ The *target component* can be identified through any of the identifiable propert
 
 If *coordinates* are used to identify the target, they must match the component fully. In other words, if __only__ *name* is given, it will __only match__ components with that name which do __not__ contain *version* or *group* fields.
 
+In *coordinates* it is also possible to provide a range of versions using the *version-range* parameter instead of *version* following the [PURL specification](https://github.com/package-url/purl-spec/blob/version-range-spec/VERSION-RANGE-SPEC.rst) as referenced by [CycloneDX](https://cyclonedx.org/docs/1.6/json/#vulnerabilities_items_affects_items_versions_items_range).
+
+The version range is provided in the schema
+
+    vers:<versioning-scheme>/<version-constraint>|<version-constraint>|...
+
+beginning with the ``vers`` identifier. Following this the versioning-scheme is specified, in the case of semantic versioning this would be ``semver`` or ``generic``. Following this a list of constraints divided by an ``|`` can be provided, to specify which versions are in scope.
+A few examples:
+
+to target all versions ``>2.0.0`` the version range to provide would be
+
+    vers:generic/>2.0.0
+
+to target all versions ``>2.0.0`` and ``<=4.5.0`` the version range to provide would be
+
+    vers:generic/>2.0.0|<=4.5.0
+
+to target all versions ``>2.0.0`` and ``<=4.5.0`` except ``4.1.1`` version range to provide would be
+
+    vers:generic/>2.0.0|<=4.5.0|>4.1.1|<4.1.1
```

Comment:
The canonical representation of this would be `vers:generic/>2.0.0|!=4.1.1|<=4.5.0`